src/Wallabag/CoreBundle/Controller/WallabagRestController.php

```diff
@@ -214,8 +213,34 @@ public function getEntriesTagsAction(Entry $entry)
      *       }
      * )
      */
-    public function postEntriesTagsAction(Entry $entry)
+    public function postEntriesTagsAction(Request $request, Entry $entry)
     {
+        $tags = explode(',', $request->request->get('tags'));
+
+        foreach ($tags as $label) {
+            $tagEntity = $this
```

Comment:
this block should be put somewhere else, maybe in the `TagRepository` (which would return a set of `Tag` objects)